plugins/catalog-backend-module-logs/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-logs",
+  "version": "0.0.0",
+  "description": "A module that subscribes to catalog releated events and logs them.",
+  "backstage": {
+    "role": "backend-plugin-module"
```

```suggestion
    "role": "backend-plugin-module",
    "pluginId": "catalog",
    "pluginPackage": "@backstage/plugin-catalog-backend"
```

since the 1.28 release, we now expect an extra set of metadata in all plugin packages - this is the effect that running `yarn fix` would get, so now hopefully you don't have to rebase and do it yourself 😅